ledger/store/trackerdb/sqlitedriver/schema.go

```diff
@@ -344,6 +344,37 @@ func accountsCreateBoxTable(ctx context.Context, tx *sql.Tx) error {
 	return nil
 }
 
+// performkvstoreNullBlobConversion scans keys with null blob value, and convert the value to `[]byte{}`.
+func performkvstoreNullBlobConversion(ctx context.Context, tx *sql.Tx) (err error) {
+	var updateWithEmptyByteSlice *sql.Stmt
+	updateWithEmptyByteSlice, err = tx.PrepareContext(ctx, "UPDATE kvstore SET value = ? WHERE key = ?")
+	if err != nil {
+		return
+	}
+	defer func() { err = updateWithEmptyByteSlice.Close() }()
+
+	var rows *sql.Rows
+	rows, err = tx.QueryContext(ctx, "SELECT key FROM kvstore WHERE value is NULL;")
+	if err != nil {
+		return
+	}
+	defer func() { err = rows.Close() }()
```

Comment:
rows.Close() is not needed either

```
// Close closes the Rows, preventing further enumeration. If Next is called
// and returns false and there are no further result sets,
// the Rows are closed automatically and it will suffice to check the
// result of Err. Close is idempotent and does not affect the result of Err.
func (rs *Rows) Close() error {
```

Reply:
see [per PR review comments, renaming method and removing db close](https://github.com/algorand/go-algorand/pull/5225/commits/3739d3f7cfeeac0e6269c5a27483a970ac756bff)